examples/auth_example.py

```diff
@@ -0,0 +1,23 @@
+# coding=utf-8
+# pylint: disable=no-name-in-module, import-error
+from lbz.authz import authorization
+from lbz.exceptions import ServerError
+from lbz.resource import Resource
+from lbz.response import Response
+from lbz.router import add_route
+
+
+class HelloWorldWithAuthorization(Resource):
+    _name = "helloworld"
+
+    @authorization
+    @add_route("/", method="GET")
+    def list(self, restrictions=None):  # pylint: disable=unused-argument
+        return Response({"message": f"Hello, {self.request.user.username} !"})
+
+
+def handle(event, context):
```

Comment:
The dev server should be a separate file but everything else could be in one.  I am not sure how
if __name__ == "__main__":
would affect lambda if it would be deployed.

Reply:
I thought they were just examples - to make them work with AWS Lambda, someone needs to put this code there anyway. Anyway, very interesting point of view that we can easily validate 🙂  But maybe in the future, right now, I can agree with separation 👍